Neos.Cache/Classes/Backend/FileBackendEntryDto.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Cache\Backend;
+
+final class FileBackendEntryDto
+{
+    private const EXPIRYTIME_LENGTH = 14;
+    private const DATASIZE_DIGITS = 10;
+
+    /**
+     * @var string
+     */
+    private $data;
+
+    /**
+     * @var string[]
+     */
+    private $tags;
+
+    /**
+     * @var int
+     */
+    private $expiryTime;
+
+
+    public function __construct(string $data, array $tags, int $expiryTime)
+    {
+        $this->data = $data;
+        $this->tags = $tags;
+        $this->expiryTime = $expiryTime;
+    }
```

Comment:
```suggestion
    public function __construct(
        private string $data,
        /** @var string[] */
        private array $tags,
        private int $expiryTime
    ) {
    }
```

Reply:
Right, I guess now with targeting 8.x we can do this, thx!